File: modul 4/PRAK404_2410817120001_Putri_Fatima_Azhara.c
#include <stdio.h>

int main() {
    while (1) {
        printf("\nPilih program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");

        int user;
        float nilai_pertama, nilai_kedua, hasil;

        printf("Masukkan Pilihan: ");
        scanf("%d", &user);

        if (user == 1) {
            printf("Masukkan nilai pertama: ");
            scanf("%f", &nilai_pertama);
            printf("Masukkan nilai kedua: ");
            scanf("%f", &nilai_kedua);
            hasil = nilai_pertama + nilai_kedua;
            printf("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f\n", nilai_pertama, nilai_kedua, hasil);
        } else if (user == 2) {
            printf("Masukkan nilai pertama: ");
            scanf("%f", &nilai_pertama);
            printf("Masukkan nilai kedua: ");
            scanf("%f", &nilai_kedua);
            hasil = nilai_pertama - nilai_kedua;
            printf("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n", nilai_pertama, nilai_kedua, hasil);
        } else if (user == 3) {
            printf("Masukkan nilai pertama: ");
            scanf("%f", &nilai_pertama);
            printf("Masukkan nilai kedua: ");
            scanf("%f", &nilai_kedua);
            hasil = nilai_pertama * nilai_kedua;
            printf("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n", nilai_pertama, nilai_kedua, hasil);
        } else if (user == 4) {
            printf("Masukkan nilai pertama: ");
            scanf("%f", &nilai_pertama);
            printf("Masukkan nilai kedua: ");
            scanf("%f", &nilai_kedua);
            hasil = nilai_pertama / nilai_kedua;
            printf("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n", nilai_pertama, nilai_kedua, hasil);
        } else if (user == 5) {
            printf("Terimakasih, telah menggunakan kalkulator Putri Fatima Az'hara\n");
            break;
        } else {
            printf("Input anda salah, silahkan coba lagi\n");
        }
    }
    return 0;
}

File: modul 4/PRAK404_2410817120001_Putri Fatima Azhara.py
while True:
    print("""
Pilih program
1. Penjumlahan
2. Pengurangan
3. Perkalian
4. Pembagian
5. Exit
""")

    try:
        user = int(input("Masukkan Pilihan: "))

        if user == 1:
            nilai_pertama = float(input("Masukkan nilai pertama: ")) 
            nilai_kedua = float(input("Masukkan nilai kedua: "))  
            hasil = nilai_pertama + nilai_kedua
            print(f"Hasil Penjumlahan antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasil:.2f}")
        elif user == 2:
            nilai_pertama = float(input("Masukkan nilai pertama: "))  
            nilai_kedua = float(input("Masukkan nilai kedua: "))  
            hasil = nilai_pertama - nilai_kedua
            print(f"Hasil Pengurangan antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasil:.2f}")
        elif user == 3:
            nilai_pertama = float(input("Masukkan nilai pertama: "))  
            nilai_kedua = float(input("Masukkan nilai kedua: "))  
            hasil = nilai_pertama * nilai_kedua
            print(f"Hasil Perkalian antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasil:.2f}")
        elif user == 4:
            nilai_pertama = float(input("Masukkan nilai pertama: "))  
            nilai_kedua = float(input("Masukkan nilai kedua: "))  
            hasil = nilai_pertama / nilai_kedua
            print(f"Hasil Pembagian antara {nilai_pertama:.2f} dengan {nilai_kedua:.2f} adalah {hasil:.2f}")
        elif user == 5:
            print("Terimakasih, telah menggunakan kalkulator Putri Fatima Az'hara")
            break
        else:
            print("Input anda salah, silahkan coba lagi")

    except ValueError:
        print("Input tidak valid, silakan masukkan angka yang sesuai.\n")